mdoc/src/main/scala/mdoc/internal/markdown/package.scala

```diff
@@ -0,0 +1,21 @@
+package mdoc.internal
+
+package object markdown {
```

Comment:
Do we need to keep it in the package object? IT seems to be only used in `MarkdownFile`, so maybe we could move it there?

Reply:
moved